packages/plugin-graphql/src/lib/menu-queries.js

```diff
@@ -0,0 +1,61 @@
+const ApolloClient = require('apollo-boost').ApolloClient;
```

Comment:
The menu is an example, please don't focus on that specifically.

Reply:
As it was a principal / defining use case from the issue, I do think it is appropriate.  If the first example can't be explored / probed / examined, I think it would possibly make the integrity of the RFC less reliable.  Now is the time to pick it all apart, IMO.

Reply:
I didn't want it as a plugin, I wanted it as core. This is 1) a demonstration of a single datatype and 2) seperated in a plugin so as to not conflict with cli for the purpose of this experiment.